README.md

```diff
@@ -371,6 +381,70 @@ Redirects user to specified view.
 await sdk.execute(Command.REDIRECT_TO, { view: View.DEALS, id: 1 });
 ```
 
+### Show floating window
+
+Opens floating window and triggers `Event.VISIBILITY` with `context` parameter that may be filled in
+and then read by event handler (see [Change visibility](#change-visibility) for details).
+
+**Parameters**
+
+| Parameter | Type   | Description                                       | Notes    |
+|-----------|--------|---------------------------------------------------|----------|
+| context   | Object | Object to be passed as JSON to event handler | optional |
```

Comment:
Is this context passing needed? Maybe for events we say that this event was triggered by the action of SDK command?

Reply:
There is an example:
```javascript
await sdk.execute(Command.SHOW_FLOATING_WINDOW, {
  context: {
    person_id: 42
  }
});
```
And then it is available in `Event.VISIBILITY` handler:
```javascript
sdk.listen(Event.VISIBILITY, ({ error, data }) => {
   data.context.person_id
});

Reply:
But this context is passed from the app iframe to the same iframe again (from command to the event). They can just skip this by keeping the state in their app.

For phone calls, we provide the context from our side to the event.

I'm fine with this, but would need to be discussed if there is actual use case here.

Reply:
> by keeping the state in their app

It is possible, but this imo giving more flexibility for non-websockets integrations.